### PR TITLE
Fixes #223

### LINKED ISSUE #223: David Thauberger Galleries
The galleries on David Thauberger's page are broken.

### PR DESCRIPTION
**Pull request by kzastrow01**

### CODE REVIEW

File: mbromley/bamboo.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
     <div class="col-lg-10 col-lg-offset-1 text-center">
       <h2>"620 Bamboo II", Michael Bromley</h2>
       <hr class="small">
-      <img style="display: block; margin: auto; width: 40%;" src="img/bamboo.JPG"/>
+      <img style="display: block; margin: auto; width: 40%;" src="img/bamboo.JPG" alt="bamboo II">
```
Comment (BmanDesignsCanada) [inline mbromley/bamboo.html]: Style should be in CSS, not html.

Comment (BmanDesignsCanada) [inline mbromley/bamboo.html]: Use lowercase extensions for filenames.

**Comment by gemohandy** [comment]
The file extensions are still uppercase. Brett said to use lowercase.

File: mbromley/bamboo.html [hunk 2]
```diff
@@ -34,7 +34,7 @@
     <div class="col-lg-10 col-lg-offset-1 text-center">
       <h2>"620 Bamboo II", Michael Bromley</h2>
       <hr class="small">
-      <img class="image" src="img/bamboo.JPG" alt="bamboo II">
+      <img class="image" src="img/bamboo.jpd" alt="bamboo II">
```
Comment (BmanDesignsCanada) [inline mbromley/bamboo.html]: WHAT IS THIS??? JPD????

**Pull request merged**